apps/credential-showcase-api-server/src/database/repositories/ScenarioRepository.ts

```diff
@@ -624,6 +643,7 @@ class ScenarioRepository implements RepositoryDefinition<Scenario, NewScenario>
       const proofRequestsResult = await tx
         .insert(ariesProofRequests)
         .values(
+          // @ts-ignore
```

Comment:
why do we need to ignore?

Reply:
The values() function does not accept undefined and the compiler complains. However you just made me realize that there was an issue in my code. Refactored